### PR TITLE
My changes

### PR DESCRIPTION
**Pull request by AsavariA**
Gallery Section

### CODE REVIEW

**Comment by PatriceJada** [comment]
Looks good to me, but why are we removing the homepage, repo URL? It would also be nice if we can get those tests working before we merge this. @AsavariA @rgrupesh

**Comment by AsavariA** [comment]
@PatriceJada you mean in the package.json file?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
   "name": "cross-polyform",
-  "homepage": "https://PolyPhyHub.github.io/PolyPhy-Website",
```
Comment (PatriceJada) [inline package.json:3]: @AsavariA Yes, why are we taking out this description?

Comment (AsavariA) [reply package.json:3]: Because its not the final URL where our site will be deployed, I will update it once we get our domain name and hosting

**Pull request merged**